public/chess.html

```diff
@@ -9,6 +9,8 @@
 
     <div id='board'></div>
 
+    <button onClick='reset()'> Reset Game </button>
```

Comment:
Consider attaching this event in a separate JS file to reduce coupling.  See [this interesting SO link](http://stackoverflow.com/questions/5871640/why-is-using-onclick-in-html-a-bad-practice) for some details.